bin/ansible-playbook

```diff
@@ -8,4 +8,4 @@ source "$(dirname "${BASH_SOURCE[0]}")/_config.sh"
 _set_openshift_env
 
 # Run ansible-playbook
-_docker_run ansible-playbook "$@"
+_docker_run ansible-playbook -e "ansible_python_interpreter=/usr/local/bin/python" "$@"
```

Comment:
Can we use an environment variable instead? I know that this configuration is not bundled with Ansible, and that we will have to handle it manually.

Reply:
You can pick ideas here: https://github.com/ansible/ansible/issues/6345

Reply:
`ansible_python_interpreter="/usr/bin/env python3"` in the inventory is the solution I choosed